src/index.ts

```diff
@@ -76,7 +76,11 @@ export const mergeDirectories = async (
 
 export default function collectFs(
   sources: string[],
-  destination: string
+  destination: string,
+  onEvent?: (data: {
```

Comment:
you could name the parameter "action", as its reminds very much of flux reducers.
```suggestion
  onEvent?: (action: {
```

Reply:
No, I don#t want to mix it up with ´redux´, because its an event callback.